pbs_light.go

```diff
@@ -26,10 +26,11 @@ var Rev string
 
 func init() {
 	rand.Seed(time.Now().UnixNano())
```

Comment:
Wondering if we need the seed initialization for the random generator in `init` or can we move that to the `main` method too

Reply:
I don't know what benefit we get from setting the seed in `init()`, but that does seem to common practice from what I could find. I can investigate further. I don't think this is related to Go 1.13.